cpu/cc13x2/ldscripts/cc1352r1.ld

```diff
@@ -0,0 +1,19 @@
+/**
+ * @addtogroup      cpu_cc13x2
+ * @{
+ *
+ * @file
+ * @brief           linker script for cc13x2 MCUs
+ *
+ * @}
+ */
+
+/* Memory Space Definitions: */
+MEMORY
+{
+    rom (rx)   : ORIGIN = 0x00000000, LENGTH = 0x58000
+    gpram      : ORIGIN = 0x11000000, LENGTH = 8K /* configurable as cache. 20K here, 8K there, and 2K in the ld-script of cc13x2ware */
+    ram (w!rx) : ORIGIN = 0x20000000, LENGTH = 20K /* sram */
```

Comment:
You can define
```Make
ROM_START_ADDR ?= 0x00000000
RAM_START_ADDR ?= 0x20000000

ROM_LEN ?= 352K
RAM_LEN ?= 20K
```

In your cpu's `Makefile.include` and the just use `cortexm_common/ldscripts/cortexm.ld` by defining `LINKER_SCRIPT ?= cortexm.ld`.